docs/flags.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Tiltfile Flags and Args
+layout: docs
+---
+
+
+Tools offer flags and args to support the wide range of users needs and preferences, and dev workflows are no different. Devs on your team will want options in how they use Tilt; some are long-standing personal preferences and others will change in the middle of working on a Pull Request. With Tilt's user configuration system, you can improve a Tiltfile that encodes one standard setup to power the breadth of workflows your team needs:
+* Maintainers choose the options to expose that make sense for their project
+* Users can easily set and change options, without having to learn a Tiltfile
+* Tiltfile extensions expose the choices so they can power arbitrary customizations
+
+This doc describes how you, the Dev Experience engineer who owns the Tiltfile, can offer this to your user, the App Developer who runs Tilt while they code. In order, we'll cover:
```

Comment:
Is this a doc or is it a guide?

Reply:
What's the difference?

Reply:
A guide is like a walk through or a tutorial, a doc describes functionality.

Reply:
I'm not familiar with this distinction; I think of doc as the parent type, and maybe there's a difference between guide and reference. Have we used this before?